test/cmdlineTests.sh

```diff
@@ -378,6 +379,12 @@ SOLTMPDIR=$(mktemp -d)
         then
             opts+=(-o)
         fi
+
+        if grep "pragma solidity ^$developmentVersion\|pragma solidity >=$developmentVersion" "$f" >/dev/null
+        then
+            echo "pragma solidity >0.0.1;" > "temp.sol"
+            grep -v "pragma solidity ^$developmentVersion\|pragma solidity >=$developmentVersion" "$f" >> "temp.sol" && mv "temp.sol" "$f"
```

Comment:
```suggestion
            grep -v "pragma solidity ^$developmentVersion\|pragma solidity >=$developmentVersion" "$f" >> "temp.sol"
            mv "temp.sol" "$f"
```

I almost missed the `mv` at the end and was going to suggest that you're leaving a temporary `temp.sol` in current directory :)
And anyway, we're running with `set -e` so it works like `&&` even when stuff is on separate lines.